java-manta-it/src/test/java/com/joyent/manta/client/multipart/EncryptedServerSideMultipartManagerIT.java

```diff
@@ -444,10 +444,10 @@ public void cantUploadSmallMultipartString() throws IOException {
             assert exception.getCause() instanceof IllegalStateException;
```

Comment:
I wouldn't use `assert` in unit tests. In some projects, I use asserts in the main code base because they aren't enabled by default. In particular for performance sensitive code paths. However, nowadays, I just tend to make them explicit checks rather than assertions.

Anyways, for this case, I would definitely use a test assertion.